services/github/github-package-json.service.js

```diff
@@ -70,6 +78,99 @@ class GithubPackageJsonVersion extends ConditionalGithubAuthService {
   }
 }
 
+class GithubPackageJsonDependencyVersion extends ConditionalGithubAuthService {
+  static get category() {
+    return 'platform-support'
+  }
+
+  static get route() {
+    return {
+      base: 'github/package-json/dependency-version',
+      pattern:
+        ':user/:repo/:kind(dev|peer)?/:scope(@[^/]+)?/:packageName/:branch*',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'GitHub package.json dependency version',
```

Comment:
Thoughts on adding an indicator here for `prod` dependency? Or is it self explanatory/implicit enough?